mtags/src/main/scala-2/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -149,15 +149,24 @@ trait MatchCaseCompletions { this: MetalsGlobal =>
       pos: Position,
       text: String
   ) extends CompletionPosition {
+    private def getSubclassesResult(tpe: Type): List[Symbol] = {
```

Comment:
```suggestion
    private def subclassesResult(tpe: Type): List[Symbol] = {
```
will be enough.

Reply:
this would cause a naming clash. There is val subclassesResult in the overridden contribute function

Reply:
`subclassesForType` maybe then ?

Reply:
updated